concordium-consensus/src/Concordium/GlobalState/Persistent/Trie.hs

```diff
@@ -57,14 +62,16 @@ class FixedTrieKey a where
 
 instance FixedTrieKey Word64
 instance FixedTrieKey Word32
-instance FixedTrieKey SHA256.Hash
+instance FixedTrieKey SHA256.Hash where
+    unpackKey (SHA256.Hash fbs) = FBS.unpack fbs
+    packKey = SHA256.Hash . FBS.pack
 deriving via SHA256.Hash instance FixedTrieKey ModuleRef
 instance FixedTrieKey AccountAddress
```

Comment:
This should also have a direct `FBS.unpack` / `FBS.pack` as in instance to remove the intermediate ByteString.

Reply:
Done.